cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/service/http/HttpHandlerGeneratorTest.java

```diff
@@ -703,7 +705,28 @@ public void dropDataset(String name) {
         public void truncateDataset(String name) {
```

Comment:
The test case is here: https://github.com/caskdata/cdap/pull/6112/files#diff-7ba8564ee62f45aad57f64190cac9e02R311